src/util/download.js

```diff
@@ -3,56 +3,33 @@ import { logWarning } from './logger.js';
 const REGEX_URL = /^(http(s?):\/\/.*)/;
 
 export function isUrl(url) {
-  return REGEX_URL.test(url);
+    return REGEX_URL.test(url);
 }
 
-export async function download (urlAddress, writablePath) {
-    // Validate input parameters
-    if (writablePath === null || writablePath === undefined || writablePath === '') {
-        throw new TypeError('Invalid writable path');
+export async function saveResource (remotePath, localPath) {
+    if (!isUrl(remotePath)) {
+        logWarning(`Remote path ${remotePath} is not a valid URL.`);
+        return remotePath;
     }
 
-    const result = REGEX_URL.exec(urlAddress);
-    if (result === null) {
+    return await download(remotePath, localPath);
```

Comment:
is localPath used at all? It looks like `download` only looks as the first param

Reply:
No, good catch @tonycuadra ! Thank you !